tests/python/test_custom_float.py

```diff
@@ -51,3 +51,22 @@ def rotate_18_degrees():
     assert x[None][0, 1] == approx(1, abs=1e-4)
     assert x[None][1, 0] == approx(-1, abs=1e-4)
     assert x[None][1, 1] == approx(0, abs=1e-4)
+
+
+@ti.test(require=ti.extension.quant)
+def test_cache_read_only():
+    ci15 = ti.type_factory_.get_custom_int_type(15, True)
+    cft = ti.type_factory_.get_custom_float_type(ci15, ti.f32.get_ptr(), 0.1)
+    x = ti.field(dtype=cft)
+
+    ti.root._bit_struct(num_bits=32).place(x)
+
+    @ti.kernel
+    def test(data: ti.f32):
+        ti.cache_read_only(x)
+        assert x[None] == data
```

Comment:
Force the `x` cache read only, to test global loading in `codegen_cuda.cpp`. Or, the `x` might be read the function in `codegen_llvm.cpp`